saleor/product/migrations/0182_remove_product_charge_taxes.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.18 on 2023-04-28 08:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("product", "0181_merge_20230417_1323"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="product",
+            name="charge_taxes",
+        ),
+    ]
```

Comment:
I think that we could remove this field only from the state in this version. 
Please make sure that this field has a default value in the database.